openmetadata-ui/src/main/resources/ui/src/components/EntityList/EntityList.tsx

```diff
@@ -92,8 +91,8 @@ export const EntityListWithAntd: FunctionComponent<AntdEntityListProp> = ({
 }: AntdEntityListProp) => {
   return (
     <Card
+      className="panel-shadow-color"
```

Comment:
If we want generic things to apply we should update it in `card.less`
so that we don't need to take care of adding this class when we use card in other places

Reply:
@chirag-madlani  Generic Css is already added to the card which can be used across the UI. Only the Left and Right panel cards have a thicker shadow that we are applying currently!